spacecmd/src/spacecmd/utils.py

```diff
@@ -607,6 +621,12 @@ def parse_str(s, type_to=None):
     >>> d = parse_str('{"channelLabel": "foo-i386-5"}')
     >>> assert d["channelLabel"] == 'foo-i386-5'
 
+        Parameters:
+            s (str): A string
+            type_to (func): A function
+
+        Returns:
+
```

Comment:
With the same idea of adding semantics to the parameters description, here we could say that `s` is the string that's going to be parsed as the type `type_to`.